### PR TITLE
Add more SF food things

### PR DESCRIPTION
**Pull request by lvn**

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -1008,14 +1008,18 @@ If you are an intern and have more info let me know.
...
-  Farolito at 24th and Mission (Cash Only).
+  Farolito at 24th and Mission (Cash Only). ([The one further down 24th](https://www.yelp.ca/biz/el-farolito-san-francisco-4) takes card.)
 * The [cioppino](https://en.wikipedia.org/wiki/Cioppino) is a San Francisco staple,
   and you can find some great cioppino in North Beach.
 * For a very satisfying and stereotypically American burger, check out Super Duper Burgers.
 * Ghirardelli's is San Francisco's famous chocolate and is worth trying.
 * Philz Coffee is everywhere and has many flavours of coffee.
 * La Mejor Bakery near 24th Mission BART station has delicious bread and pastries baked daily, well worth checking out.
 * Pineapple King is an awesome Hong Kong style bakery in the Sunset. Try their butter pineapple bun!
+* There's not many good AYCE KBBQ places in SF - [Carbon Grill](https://www.yelp.ca/biz/carbon-grill-san-francisco-3) is probably the best one.
```
Comment (tonyd3) [inline index.markdown]: nit: "There are not many"

File: index.markdown [hunk 2]
```diff
@@ -1008,14 +1008,18 @@ If you are an intern and have more info let me know.
...
+* The [SoMa StrEAT Food Park](http://www.somastreatfoodpark.com/) has a rotating selection of (decently good) food trucks.
+* SF is the world capital of hipster coffee. Some of the more well-known ones include [Sightglass](https://sightglasscoffee.com/), [Four Barrel](http://fourbarrelcoffee.com/), [Ritual](https://www.ritualroasters.com/), [Mazarine](http://www.mazarinecoffee.com/), [Andytown](http://www.andytownsf.com/)...
```
Comment (tonyd3) [inline index.markdown]: nit: "hipster coffee" -> "coffee"

**Comment by tonyd3** [comment]
LGTM, Thanks!

**Comment by lvn** [comment]
@tonyd3 Fixed nits.

**Comment by tonyd3** [comment]
thanks!

**Pull request merged**